src/main/java/com/epam/team1/emotional/help/service/QuizService.java

```diff
@@ -47,4 +53,14 @@ public QuizResponseDto prepareQuizResponseDto(Result result, Quiz quiz){
         dto.setTotalPoints(quiz.getTotalPoints());
         return dto;
     }
+    public void sendResultOfQuizToEmailAndSaveIt(SendQuizResultToEmailDto dto){
+        mailService.sendQuizResultToMail(dto);
+        try {
```

Comment:
Try to get rid of the `try/catch` block here + try to specify the narrower exception class in the future.